coupon_commercial_partner_applicability/models/coupon_coupon.py

```diff
@@ -3,12 +3,13 @@
 from odoo import _, models
 
 
-class SaleCoupon(models.Model):
-    _inherit = "sale.coupon"
+class Coupon(models.Model):
+    _inherit = "coupon.coupon"
 
-    def _check_coupon_code(self, order):
+    def _check_coupon_code(self, order_date, partner_id, **kwargs):
         """All the partners in the commercial entity can use their common coupons"""
-        message = super()._check_coupon_code(order)
+        message = super()._check_coupon_code(order_date, partner_id, **kwargs)
+        order = kwargs.get("order")
```

Comment:
There's no need to get the order. Check how the base method does it and adapt to it

Reply:
done, thanks :)